### DISCUSSION TITLE
How logger works?

### DISCUSSION THREAD

**Discussion by SalmenBejaoui** [algolia/instantsearch#5992] [Q&A]
I'm trying to log Algolia queries. The logger does not work because it never reaches breakpoints.

<img width="819" alt="image" src="https://github.com/algolia/instantsearch/assets/30017004/9a422e2d-7b14-44fc-8975-8f4a0652545a">

**Comment by dhayab** [comment]
Hi, you can take a look at the documentation on how to customize logging for the JavaScript search client here: https://www.algolia.com/doc/api-client/getting-started/customize/javascript/?client=javascript#logging.

If you still have questions, you can reach out to the maintainers at this page at [algolia/algoliasearch-client-javascript](https://github.com/algolia/algoliasearch-client-javascript).

**Reply by SalmenBejaoui** [reply]
> Hi, you can take a look at the documentation on how to customize logging for the JavaScript search client here: https://www.algolia.com/doc/api-client/getting-started/customize/javascript/?client=javascript#logging.
> 
> 
> 
> If you still have questions, you can reach out to the maintainers at this page at [algolia/algoliasearch-client-javascript](https://github.com/algolia/algoliasearch-client-javascript).

I tried to go with my own implementation respecting the Logger interface to be able to share data with external Log system but it doesn't work.
